### PR TITLE
Create /solutions/telco/5g-edge webpage

### PR DESCRIPTION
**Pull request by britneywwc**
## Done

- Created /solutions/telco/5g-edge page
- Similar to the /solutions/telco/5g-core page
  - Minor fixes to this page for the equal height row items

## QA

- Go to https://canonical-com-1383.demos.haus/solutions/telco/5g-edge
- Check that design matches against [Figma](https://www.figma.com/design/brgzRViEncgKKD4KzVnyXy/24.10-Canonical.com%2Fsolutions?node-id=440-6534&node-type=frame&m=dev) and content matches [copy doc](https://docs.google.com/document/d/1K_aHQNapkDgLdCNpLsB1oEeHxo68TMDJoU-iDYlI7lU/edit)

## Issue / Card

Fixes [WD-14131](https://warthogs.atlassian.net/browse/WD-14131)

## Screenshots

[if relevant, include a screenshot]

[WD-14131]: https://warthogs.atlassian.net/browse/WD-14131?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by britneywwc** [comment]
@Sophie-32 I'm using the same form modal for 5g-edge and 5g-core since there's no explicit instructions in this copy doc to create a new modal. Let me know if this should be changed.

**Comment by mattea-turic** [comment]
V v small thing, could you add break after "future"?
Screenshot 2024-10-01 at 14 28 48

Break after "future"
Screenshot 2024-10-01 at 14 29 22

This section was an accidental duplicate on the figma, so could you delete this one (the one w/o image) pls
Screenshot 2024-10-01 at 14 39 47

everything else is good, so I'll +1 with changes

File: templates/solutions/telco/5g-edge.html [hunk 1]
```diff
@@ -0,0 +1,809 @@
...
+          for 5G Edge infrastructure
+        </h2>
+      </div>
+      <div class="col">
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Real-time performance to minimise latency</li>
+          <li class="p-list__item is-ticked">Robust security features</li>
+          <li class="p-list__item is-ticked">Seamless scalability</li>
+          <li class="p-list__item is-ticked">Strong vendor partnerships</li>
+          <li class="p-list__item is-ticked">Compliant open source with 24x7 support</li>
```
Comment (petesfrench) [inline templates/solutions/telco/5g-edge.html:133]:
```suggestion
          <li class="p-list__item is-ticked">Compliant open source with 24/7 support</li>
```

File: templates/solutions/telco/5g-edge.html [hunk 2]
```diff
@@ -0,0 +1,809 @@
...
+                    hi_def=True,
+                    loading="lazy",
+                    attrs={"class": "p-image-container__image"}) | safe
+          }}
+        </div>
+      </div>
+      <div class="col">
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">
+            <span class="p-heading--5">Ultra-low latency:</span>Ensure real-time responsiveness.
```
Comment (petesfrench) [inline templates/solutions/telco/5g-edge.html:166]:
```suggestion
            <span class="p-heading--5">Ultra-low latency:</span> Ensure real-time responsiveness.
```

Comment (petesfrench) [reply templates/solutions/telco/5g-edge.html:166]: This and the rest in this list need spaces between the colon and following word

**Comment by petesfrench** [comment]
I am fairly sure this grey text on grey background wont pass accessibility checks. Can we use colours with greater contrast @mattea-turic?
image

**Comment by petesfrench** [comment]
The section 'Maximise efficiency with Canonical MicroCloud
for constrained environments' appears to be duplicated

**Comment by petesfrench** [comment]
Can you had `/5g-edge` to the sitemap. And if `/5g-core` isn't there, add it too

File: templates/solutions/telco/5g-edge.html [hunk 3]
```diff
@@ -0,0 +1,809 @@
...
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="p-section">
+    <div class="u-fixed-width">
+      <div class="p-section--shallow">
+        <hr class="p-rule--highlight" />
+        <h2 class="p-text--small-caps">Trusted by leading telco providers</h2>
```
Comment (petesfrench) [inline templates/solutions/telco/5g-edge.html:659]:
```suggestion
        <h2 class="p-text--small-caps">See what industry players have to say</h2>
```
Is what it says in the doc, not sure which is right though

File: templates/solutions/telco/5g-edge.html [hunk 4]
```diff
@@ -0,0 +1,809 @@
...
+      <div class="col-9">
+        <div class="p-section--shallow">
+          <div class="row">
+            <div class="col-3">
+              <h3 class="p-heading--5">Webinars</h3>
+            </div>
+            <div class="col-6">
+              <ul class="p-list--divided">
+                <li class="p-list__item">
+                  <a href="https://ubuntu.com/engage/telco-cloud-implementation">Telco cloud implementation</a>
```
Comment (petesfrench) [inline templates/solutions/telco/5g-edge.html:739]:
```suggestion
                  <a href="https://ubuntu.com/engage/telco-cloud-implementation">Telco cloud implementation with open source</a>
```

**Review by petesfrench — COMMENTED** [review]
This is only half the review, I wont have time to finish today, sorry. Will pick it up tomorrow morning

File: templates/solutions/telco/5g-edge.html (hunk 3 above)
Comment (britneywwc) [reply templates/solutions/telco/5g-edge.html:659]: Changed it to match the copy docs

**Comment by mattea-turic** [comment]
> I am fairly sure this grey text on grey background wont pass accessibility checks. Can we use colours with greater contrast @mattea-turic? image

This meets wcag aa which is the level we're aiming for, so it should pass accessibility checks

File: templates/solutions/telco/5g-edge.html [hunk 5]
```diff
@@ -0,0 +1,722 @@
...
+  </div>
+
+  <div class="p-section">
+    <div class="u-fixed-width">
+      <div class="p-section--shallow">
+        <hr class="p-rule--highlight" />
+        <h2 class="p-text--small-caps">See what industry players have to say</h2>
+      </div>
+    </div>
+    {%- call(slot) quote_wrapper(
```
Comment (petesfrench) [inline templates/solutions/telco/5g-edge.html:575]: nice

File: templates/solutions/telco/5g-edge.html [hunk 6]
```diff
@@ -0,0 +1,722 @@
...
+        <p>
+          <a href="/contact-us"
+             class="p-button--positive"
+             aria-controls="telco-5g-core-modal">Contact us</a>
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <script defer src="{{ versioned_static('js/modals.js') }}"></script>
```
Comment (petesfrench) [inline templates/solutions/telco/5g-edge.html:715]: Perhaps 'modals' could be added to build

File: templates/solutions/telco/5g-edge.html [hunk 7]
```diff
@@ -0,0 +1,722 @@
...
+
+  <div class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Be ready for the future <br /> of your 5G network</h2>
+      </div>
+      <div class="col">
+        <div class="p-image-container">
+          {{ image(url="https://assets.ubuntu.com/v1/91a57c15-chart-placeholder.png",
```
[the remark this answers is not in the document]
Comment (britneywwc) [reply templates/solutions/telco/5g-edge.html:313]: Not that I'm aware of from the design and copy docs. @mattea-turic @Sophie-32 can you please weigh in on this?

**Comment by Sophie-32** [comment]
> @Sophie-32 I'm using the same form modal for 5g-edge and 5g-core since there's no explicit instructions in this copy doc to create a new modal. Let me know if this should be changed.

There's a comment in the copydoc with a link to a form, can you please double check if this is a new one? - https://docs.google.com/document/d/1mi6NOW_GiCMZ1O4xRCxS2RPhZMjEaOdipD5LFnCvMz4/edit

File: templates/solutions/telco/5g-edge.html (hunk 7 above)
[the remark this answers is not in the document]
Comment (Sophie-32) [reply templates/solutions/telco/5g-edge.html:313]: I haven't gotten this kind of question before. Perhaps you can elaborate? Is the problem that images usually have more descriptive names?

**Pull request merged**